src/components/Login/Login.module.scss

```
@@ -0,0 +1,136 @@
+@import 'src/style/themes/default/colors';
+
+.loginPage {
```

Tu pourrais encapsuler tes classes en fonction de leur hierarchie dans ton code.

Comme par exemple:

```suggestion
.loginPage {
    display: flex;
    flex-direction: column;
    height: 100%;
    width: 100%;
    justify-content: space-between;
    padding: 24px;

    .loginContainer {
        display: flex;
        flex-direction: column;
        justify-content: center;
        background-color: $cyan-7;
        max-width: 833px;
        margin: auto;
    }
    [...]
 }
```